### DISCUSSION TITLE
feh ERROR: Can't open X display

### DISCUSSION THREAD

**Discussion by gerar3** [termux/termux-packages#12437] [Q&A]
I installed the feh package and when I use it:
feh fig2.png => feh ERROR Can't open X display
however the following packages are installed:
x11-repo - xwayland - xorg-server
and I put export DISPLAY=:0 in the .bashrc file
I don't know how can I solve this problem.
Thanks fora help.

**Comment by 2096779623** [comment]
> and I put export DISPLAY=:0 in the .bashrc file

Restarted termux?

**Comment by sylirre** [comment]
Just putting `export DISPLAY=:0` to your bashrc file is not enough.

The changes made to `~/.bashrc` take effect on next shell launch. You need to restart Termux app, that's the first thing required. Another one is to check whether your X server is actually running and uses this display port.

**Reply by gerar3** [reply]
Thanks, yes I restarted termux after modifying the .bashrc file.
Regarding the X server how to check that it is active and find which port it uses ?
